SECURITY_CONSIDERATIONS.md

```diff
@@ -0,0 +1,6 @@
+# Security Considerations
+
+OpenAPI documents use JSON, YAML and JSON Schema and therefore share their same security considerations. [JSON Schema Core](https://json-schema.org/draft/2020-12/json-schema-core#section-13) [JSON Schema Validation](https://json-schema.org/draft/2020-12/json-schema-validation#name-security-considerations) [YAML](https://www.ietf.org/archive/id/draft-ietf-httpapi-yaml-mediatypes-10.html) [JSON](https://www.rfc-editor.org/rfc/rfc8259)
```

Comment:
```suggestion
OpenAPI documents use JSON, YAML, and JSON Schema, and therefore share their security considerations:
- [JSON](https://www.rfc-editor.org/rfc/rfc8259)
- [YAML](https://www.ietf.org/archive/id/draft-ietf-httpapi-yaml-mediatypes-10.html)
- [JSON Schema Core](https://json-schema.org/draft/2020-12/json-schema-core#section-13)
- [JSON Schema Validation](https://json-schema.org/draft/2020-12/json-schema-validation#name-security-considerations)
```

Reply:
@ralfhandl the [RFC 9215: YAML Media Type](https://datatracker.ietf.org/doc/html/rfc9512) is out! 🎉